packages/steiger/src/cli.ts

```diff
@@ -88,16 +87,31 @@ const yargsProgram = yargs(hideBin(process.argv))
 
 const consoleArgs = yargsProgram.parseSync()
 
-const targetPath = resolve(consoleArgs._[0])
-
-try {
-  if (!(await stat(targetPath)).isDirectory()) {
-    console.error(`${consoleArgs._[0]} is a file, must be a folder`)
-    process.exit(102)
+let targetPath: string | undefined
+if (consoleArgs._.length > 0) {
+  if (await existsAndIsFolder(consoleArgs._[0])) {
```

Comment:
`consoleArgs._[0]` is not very readable and is used in several places, can we have a variable like `inputPath` for it at the top?